img_cmd_training_new_dataset.py

```diff
@@ -0,0 +1,280 @@
+from argparse import ArgumentParser
+
+import matplotlib.pyplot as plt
+import os.path
+from os import path
+
+from fastai.vision.all import *
+from fastai.callback.progress import CSVLogger
+
+import wandb
+
+# Initialize W&B run (if not already initialized)
+run = wandb.init(
+    project="third-testing",
+    entity="vuhachau2412",
+    notes="A set of small/useless datasets for testing.",
+    job_type="dataset-upload"
+)
+# Load the dataset artifact
+artifact = run.use_artifact("arcslaboratory/Multirun-testing-1K+/larger-perfect-dataset:v0")
+artifact_dir = artifact.download()
+
+
+dataset_path = artifact_dir + '/'   # Path to the extracted images from the artifact
+
+from torch.utils.data import Dataset
+
+# Constants 
+# Load the dataset artifact
+
+VALID_PCT = 0.05
+NUM_REPLICATES = 2
+NUM_EPOCHS = 1
+# DATASET_DIR = Path("/data/clark/summer2021/datasets")
+DATASET_DIR = Path(dataset_path)
+MODEL_PATH_REL_TO_DATASET = Path("cmd_models_fixed")
+DATA_PATH_REL_TO_DATASET = Path("cmd_data_fixed")
+# VALID_MAZE_DIR = Path("../Mazes/validation_mazes8x8/")
+
+compared_models = {
+    "resnet18": resnet18,
+    "resnet34": resnet34
+}
+
+DATASET_DIR.ls()
+
+def filename_to_class(filename: str) -> str:
```

Comment:
This reminds me: in `boxnav` we should always give the angle and not set it to zero since we handle that here.